.github/kokoro/ice40.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+CALLED=$_
+[[ "${BASH_SOURCE[0]}" != "${0}" ]] && SOURCED=1 || SOURCED=0
```

Comment:
nit: Prefer `if` block over short-circuit assignment. (Maybe also from recipe or example?)

Reply:
I think it comes from https://stackoverflow.com/questions/2683279/how-to-detect-if-a-script-is-being-sourced
> ```shell
> [[ $_ != $0 ]] && echo "Script is being sourced" || echo "Script is a subshell"
> ```

Reply:
It's unclear if we need the SOURCED variable at all.  Should we just remove this?